src/GitInfo/build/GitInfo.targets

```diff
@@ -339,7 +347,7 @@
 			<GitIsDirty>0</GitIsDirty>
 		</PropertyGroup>
 
-		<WriteLinesToFile File="$(_GitIsDirtyFile)" Lines="$(GitIsDirty)" WriteOnlyWhenDifferent="true"/>
+		<WriteLinesToFile File="$(_GitIsDirtyFile)" Lines="$(GitIsDirty)" WriteOnlyWhenDifferent="true" Overwrite="true"/>

```

Comment:
Overwrite also be solved here: https://github.com/devlooped/GitInfo/pull/184

Reply:
That PR has been merged now. Hopefully the update will be trivial for this one.